Add example of child module outputs in terraform output CLI reference (#1163)
hashicorp/web-unified-docs: content/terraform/v1.13.x/docs/cli/commands/output.mdx

]
```

The `terraform output` command only displays outputs from the root module. 

> To display outputs from child modules, you must define a new output in the 

Reviewer: ```suggestion
To display outputs from child modules, define an `output` block in your 
```

root module with a value of the child submodule output.

```hcl
module "random" {
  source = "./modules/random"
}

> output "string" {

Reviewer: I'm sorry to be that guy, but can we make this a little more realistic? Like make it an IP address or something?
